deploy/openshift-clusters/scripts/baremetal-wizard.sh

```
@@ -0,0 +1,569 @@
+        fi
+        echo "${line}" >> "${tmp_inventory}"
+    done
+
+    {
+        echo ""
+        echo "[baremetal_nodes:vars]"
+        echo "bmc_driver=redfish"
+        echo "bmc_port=443"
+        echo "bmc_verify_ca=False"
+        echo "cpu_arch=x86_64"
+    } >> "${tmp_inventory}"
```

Shellcheck (0.11.0)

[style] Consider using { cmd1; cmd2; } >> file instead of individual redirects.

(SC2129)